src/main/java/dev/j3fftw/litexpansion/ItemSetup.java

```diff
@@ -48,17 +53,21 @@ private void registerTools() {
 
     private void registerMachines() {
         new FoodSynthesizer().register(LiteXpansion.getInstance());
-
         new ScrapMachine().register(LiteXpansion.getInstance());
         new MassFabricator().register(LiteXpansion.getInstance());
-        new RubberSynthesizer().register(LiteXpansion.getInstance());
         new RefinedSmeltery().register(LiteXpansion.getInstance());
     }
 
-    private void registerMiscItems() {
-        //Rubber
-        registerNonPlaceableItem(Items.RUBBER, RubberSynthesizer.RECIPE_TYPE, SlimefunItems.OIL_BUCKET);
+    //Disable when SlimyTreeTaps exists
+    private void registerRubber() {
+        if (!getServer().getPluginManager().isPluginEnabled("SlimyTreeTaps")) {
```

Comment:
Should rather check "isPluginEnabled()" instead of a null check here. xx cookie

Reply:
> you also need to soft depend it in the plugin.yml

Its already soft depended, you guys added it a while ago.